LICENSE.txt

```diff
@@ -0,0 +1,13 @@
+Modified BSD License
```

Comment:
I suggest that we consider this a prototype, meaning that we can throw out all of the code if necessary, and put something reasonable here as a placeholder so we can work.